src/components/Wallet/BalanceOverview/index.js

```diff
@@ -45,17 +52,23 @@ function BalanceOverview({
         <PolitipoolLLMModalWrapper key="pool" />,
       ],
     },
-    {
-      amount: formatDollars(balances.liquidAmount.amount),
-      title: 'Liquid LLD',
-      currency: 'LLD',
-      icon: LLD,
+    showStaked && {
```

Comment:
why is there a showstaked condition? Whats it supposed to be doing and why?

Reply:
not show staked coins for congress and senate wallets